docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -13,9 +13,16 @@ This section is an extension of the main [column definitions documentation](/com
 
 ### Value getter
 
-Sometimes a column might not have a corresponding value and you just want to render a combination of different fields. To do that, you can set the `valueGetter` attribute of `ColDef` as in the example below:
+Sometimes a column might not have a corresponding value and you just want to render a combination of different fields.
+To do that, you can set the `valueGetter` attribute of `ColDef` as in the example below:
```

Comment:
A thought, we could have hidden the feature and encourage a transformation on the input rows instead. Same outcome.